_assets/stylesheets/pages/social.scss

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
not a huge deal, but i'd rather have this in /modules or a new one like /partials